Production_scripts/Process_extracts/Produce_extract_tests/extract_PIS_tests.R

```diff
@@ -0,0 +1,46 @@
+####################################################
+# Name of file - PIS-tests.R
+# Original Authors - Jennifer Thom
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               PIS processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "PIS", ext = "rds"))
+
+# Read current SLF episode file and filter for 01B and GLS records
+existing_data <- get_existing_data_for_tests(new_data = new_data)
+
+
+# Produce comparison-------------------------------------
+# Compare new file with existing slf data
+comparison <- produce_test_comparison(
+  produce_source_pis_tests(existing_data,
+    sum_mean_vars = c("cost", "no_dispensed_items")
+  ),
+  produce_source_pis_tests(new_data,
+    sum_mean_vars = c("cost", "no_dispensed_items")
+  )
+)
```

Comment:
Since this is a custom function for PIS, I think we don't need to specify the variables like this, and can just 'hard-code' them into the `produce_source_pis_tests` function. In fact these are already the default, so this will work as is.

```suggestion
comparison <- produce_test_comparison(
  produce_source_pis_tests(existing_data),
  produce_source_pis_tests(new_data)
)
```